Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -214,9 +220,10 @@ class BrowserViewController: UIViewController {
         applyTheme()
     }
 
-    @objc
-    func didTapUndoCloseAllTabToast(notification: Notification) {
-        leaveOverlayMode(didCancel: true)
+    /// If user manually opens the keyboard and presses undo, the app switches to the last
+    /// open tab, and because of that we need to leave overlay state
+    @objc func didTapUndoCloseAllTabToast(notification: Notification) {
```

Comment:
I think the @objc was moved on the line above in main, this could cause a swiftlint warning?